tensorflow/lite/tools/cmake/test_utils/run-tests.cmake

```diff
@@ -0,0 +1,7 @@
+if(NOT DEFINED ENV{TESTS_ARGUMENTS})
```

Comment:
Could you add the following TensorFlow license notice?
```
#
# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      https://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
Done & squashed the commits